.vagrant/machines/imcs/virtualbox/action_provision

```diff
@@ -0,0 +1 @@
+1.5:87b3f301-8382-4277-ab67-7e864df4a869
```

Comment:
@olivermel - Still need to remove these .vagrant/* files.  You can remove these from the repository (without deleting them on your machine), but running this command:
```
git rm --cached -r .vagrant
```